### DISCUSSION TITLE
Neuralcoref with Spacy 3.0

### DISCUSSION THREAD

**Discussion by morankyle** [explosion/spaCy#8387] [Help: Installation] [answered]
Am I able to use neuralcoref without cloning the Git repo? Can I use the source file on PyPI listed below? Am currently using Python 3.0 so am unable to use pip.

https://files.pythonhosted.org/packages/0c/40/8db3db763077fe80b71859f57731261aeb03cc624635f97a3bcfe55ab37b/neuralcoref-4.0.tar.gz

**Comment by polm** [comment, marked as answer]
NeuralCoref does not support spaCy v3, you'll need to use v2. We're working on a new coref component for v3, and while we don't have a planned release date we hope to be able to release it soon.

To be clear, are you literally using Python 3.0, as opposed to 3.7 or something? spaCy and most other packages won't work on that.

Assuming that link is to a PyPI package, which is what it looks like, you should be able to install it with pip. It may be possible to install it in other ways, like unzipping it and putting it somewhere, but I would strongly recommend against that and would expect it to cause many problems.

**Reply by morankyle** [reply]
My apologies, typo as I'm using Spacy 3.7. Thats to hear about the neuralcoref being released for v3 soon!

**Reply by svlandeg** [reply]
Update: our coref component is now released in `spacy-experimental`, cf https://github.com/explosion/spaCy/discussions/11585!
